apps/web/src/components/Wordmark/Wordmark.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import Icon, { iconColor } from '../Icon/Icon';
+import { ReactComponent as RedditWordmark } from '../../assets/redditWordmark.svg';
+
+/**
+ * Renders a logo component within a navbar.
+ *
+ * @returns {React.ReactElement} A JSX element representing the Reddit Wordmark component.
+ */
+
+const Wordmark = (): React.ReactElement => {
+  return (
+    <header>
```

Comment:
I think semantically this makes more sense as a heading `<h1>` vs. a `<header>`.

This is a good time to remind you about Semantic HTML . Always remember that each tag means something and its important for accessibility to try to pick the right HTML element whenever possible. Ask yourself "What does this piece of UI that I'm building do?" and "Is there an HTML tag that does that thing?". What is an `<h1>` for? What about a `<header>`?